features/admins/users/admin_find_users.feature

```diff
@@ -11,69 +11,98 @@ Feature: Admin Find Users page
       And I am logged in as a super admin
       And I go to the manage users page
 
-  Scenario: The Find Users page should perform a partial match on name
-    When I fill in "Name" with "user"
+  Scenario: The Find Users page performs a partial match on name with * wildcard
+    When I fill in "Name" with "u*er*"
       And I submit
     Then I should see "userA"
       And I should see "userB"
       And I should see "userCB"
 
-  Scenario: The Find Users page should perform a exact match on name if exact is checked
-    When I check "exact"
-      And I fill in "Name" with "user"
+  Scenario: The Find Users page performs a exact match on name by default
+    When I fill in "Name" with "user"
       And I submit
     Then I should see "0 users found"
     When I fill in "Name" with "userA"
       And I submit
     Then the field labeled "user_email" should contain "a@ao3.org"
       But I should not see "UserB"
 
-  Scenario: The Find Users page should perform a partial match by email
-    When I fill in "Email" with "bo3"
+  Scenario: The Find Users page searches past logins only if the option is selected
```

Comment:
Could you add a test (not necessarily here, but I think it needs to be a feature test) for admins without the right role(s) not having the checkbox? And a test that if someone passes it anyway, the option is ignored? (I think the second would be easier as a controller test.)